src/components/AssetManagementPanel/AssetManagementForm/FormVariants/TransferHolderForm/TransferHolderForm.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import { ButtonSolidOrangeWhite, ButtonSolidWhiteGrey } from "../../../../UI/Button";
+import { AssetInformationPanel } from "../../../AssetInformationPanel";
+import { AssetManagementActions } from "../../../AssetManagementActions";
+import { AssetManagementTitle } from "../../AssetManagementTitle";
+import { EditableAssetTitle } from "./../EditableAssetTitle";
+
+interface TransferHolderProps {
+  formAction: AssetManagementActions;
+  onSetFormAction: (nextFormAction: AssetManagementActions) => void;
+  tokenId: string;
+  tokenRegistryAddress: string;
+  beneficiary?: string;
+  holder?: string;
+  handleTransfer: (newHolder: string) => void;
+  holderState: string;
+}
+
+export const TransferHolderForm = ({
+  formAction,
+  onSetFormAction,
+  tokenId,
+  tokenRegistryAddress,
+  beneficiary,
+  holder,
+  handleTransfer,
+  holderState,
+}: TransferHolderProps) => {
+  const [newHolder, setNewHolder] = React.useState<string>("");
+
+  const onBackHandler = () => {
+    onSetFormAction(AssetManagementActions.None);
+  };
+
+  const onSetNewHolder = (value: string) => {
+    setNewHolder(value);
+  };
+
+  const onHandleTransfer = () => {
+    handleTransfer(newHolder);
+  };
+
+  const validateTransfer = () => {
+    if (!newHolder) return true;
+    if (newHolder === holder) return true;
```

Comment:
why should the newHolder be the current holder?

Reply:
from the understanding from Boon, he said that if the user keys in the same holder number as the current holder, we do not want to let him proceed, thus disable the transfer button.

Reply:
ohhhhhhhh this is super funky.... `return true = not valid?`

can we either change this fucntion to `isValidTransfer` if you intend to return T/F (at the same time, t=valid, f = invalid) or `validateTransfer` if you just throw error if it doesnt validate, and it returns nothing.